book/src/bank-forks.md

```diff
@@ -11,8 +11,9 @@ challenges:
 
 ## Architecture
 
-The basic design idea is to maintain a DAG of forks. The DAG is initialized with
-a *root*.  Each subsequent fork must descend from the root.
+The basic design idea is to maintain a DAG of Banks checkpointed at different
```

Comment:
Implemenation is that BankForks maintains an active set of forks, and the DAG can be derived by traversing from the active set to the parents of each active bank.